### DISCUSSION TITLE
System tray support for desktop app

### DISCUSSION THREAD

**Discussion by pastellexists** [stoatchat/discussions#217] [Feature Suggestions]
(Speaking from a windows perspective)

Exiting out of the Revolt desktop app (just "Revolt" from here on out) should minimize the application to the system tray, where the icon can be clicked to bring it back. Note that when I say system tray, I do not mean the taskbar. I mean the little collection of icons to the right of it, including wifi connections and audio, adjacent to the date and time.

The context menu of this system tray icon only needs a quit button, which would actually exit the application.

**Comment by ghost** [comment]
I have this issue, but in Linux Mint Cinnamon. The system tray icon do not exists for me, and closing the client will literally close the whole client down.

**Reply by pastellexists** [reply]
To the best of my knowledge, that is the intended behavior at the moment. I'm suggesting that the behavior be changed.

**Reply by cyrneko** [reply]
that's exactly the issue hat Pastell wants to see fixed, _there is no tray icon for Revolt at the moment_

**Comment by exdnaa** [comment]
That is a great idear.
